docs/rfcs/XXX-pull-request-review.md

```diff
@@ -0,0 +1,151 @@
+# Feature title
+
+Pull Request Review
+
+## Status
+
+Proposed
+
+## Summary
+
+Give and receive code reviews on pull requests within Atom.
+
+## Motivation
+
+Workflows around pull request reviews involve many trips between your editor and your browser. If you check out a pull request locally to test it and want to leave comments, you need to map the issues that you've found in your working copy back to lines on the diff to comment appropriately. Similarly, when you're given a review, you have to mentally correlate review comments on the diff on GitHub with the corresponding lines in your local working copy, then map _back_ to diff lines to respond once you've established context. By revealing review comments as decorations directly within the editor, we can eliminate all of these round-trips and streamline the review process for all involved.
+
+Peer review is also a critical part of the path to acceptance for pull requests in many common workflows. By surfacing progress through code review, we provide context on the progress of each unit of work alongside existing indicators like commit status.
+
+## Explanation
+
+### Current pull request tile
+
+Reviews on the current pull request are rendered as a list on the current pull request tile.
+
+![review-list](https://user-images.githubusercontent.com/378023/46273505-b9533280-c590-11e8-840e-a8eac8023cad.png)
+
+* The review summary bubble is elided after the first sentence or N characters if necessary.
+* Clicking the review summary bubble opens an `IssueishDetailItem` in the workspace center.
+* Clicking a line comment opens or activates an editor on the referenced file and scrolls to center the comment's line, translated according to local changes if appropriate.
+* Line comments within the review are rendered: _with a vertical blue bar_ before the file has been opened and the corresponding decoration is visible; _with a vertical grey bar_ after the file and decoration have been seen; and _with a vertical green bar_ after the comment has been marked "resolved" with the control on its decoration.
+* The review summary bubble and line comment lists are greyed out if a different `IssueishDetailItem` is activated.
+
+If a pending review is present, it appears at the top of the "Reviews" section within this tile:
+
+![pending-review](https://user-images.githubusercontent.com/378023/46275946-9bd69680-c599-11e8-9889-66c35458286a.png)
+
+* The review summary is a TextEditor that may be used to compose a summary comment.
+* Choosing "Cancel" dismisses the review and any comments made. If there are local review comments that will be lost, a confirmation prompt is shown first.
+* Choosing "Submit review" submits the drafted review to GitHub.
+
+If there is no pending review, the tile instead displays a control to create one.
+
+### IssueishDetailItem
```

Comment:
@kuychaco One big question based on this section. Where do we put the conversation timeline?